icons/salad.svg

```diff
@@ -9,9 +9,9 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="M7 21h10" />
-  <path d="M12 21a9 9 0 0 0 9-9H3a9 9 0 0 0 9 9Z" />
-  <path d="M11.38 12a2.4 2.4 0 0 1-.4-4.77 2.4 2.4 0 0 1 3.2-2.77 2.4 2.4 0 0 1 3.47-.63 2.4 2.4 0 0 1 3.37 3.37 2.4 2.4 0 0 1-1.1 3.7 2.51 2.51 0 0 1 .03 1.1" />
-  <path d="m13 12 4-4" />
-  <path d="M10.9 7.25A3.99 3.99 0 0 0 4 10c0 .73.2 1.41.54 2" />
+  <path d="M7 21h10"/>
+  <path d="M3 12a9 9 0 0 0 18 0H3"/>
+  <path d="M11.38 12A2.4 2.4 0 0 1 11 7.23a2.4 2.4 0 0 1 3.18-2.73 2.4 2.4 0 0 1 3.47-.67A2.4 2.4 0 0 1 21 7.2a2.4 2.4 0 0 1-1.08 3.7 2.5 2.5 0 0 1 .03 1.1"/>
+  <path d="m13 12 4-4"/>
+  <path d="M10.9 7.25A4 4 0 0 0 4.5 12"/>
```

Comment:
We seem to be losing some detail here (green line original):
![image](https://user-images.githubusercontent.com/17746067/232280781-a626278e-519f-470a-ae50-4bc110871e07.png)